### PR TITLE
Update metadata content

### LINKED ISSUE #757: Update metadata
image

Notice lines 8, 13, and 20 - Let's remove the duplication and change it to "Give to Thunderbird" (line 21 should be updated too). We should keep "Donate to Mozilla" on the MoFo version of the site.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #757

### CODE REVIEW

File: maintenance/maintenance.html [hunk 1]
```diff
@@ -4,17 +4,17 @@
 <html lang="en-US">
     <head>
         <meta charset="utf-8">
-        <title>Donate to Thunderbird | Give to Thunderbird</title>
+        <title>Give to Thunderbird</title>
```
Comment (mmmavis) [inline maintenance/maintenance.html]: Is this maintenance page only for Thunderbird site though?

Comment (mmmavis) [reply maintenance/maintenance.html]: If not we should open a follow-up ticket so we can have a Foundation version of maintenance page.

File: donate/thunderbird/templates/base.html [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 {% load i18n %}
 
-{% block title_prefix %}{% if request.site and page.pk == request.site.root_page.pk and request.site.site_name %}{% trans 'Donate to Thunderbird' context 'Page title' %} | {% endif %}{% endblock %}{% block title %}{% if page.seo_title %}{{ page.seo_title }}{% else %}{{ page.title }}{% endif %}{% endblock %}{% block title_suffix %}{% if request.site and page.pk != request.site.root_page.pk and request.site.site_name %} | {% trans 'Donate to Thunderbird' context 'Page title' %}{% endif %}{% endblock %}
+{% block title_suffix %} | {% trans 'Donate to Mozilla' context 'Page title' %}{% endblock %}
```
Comment (TheoChevalier) [inline donate/thunderbird/templates/base.html]: I think you want "Give to Thunderbird" here?

Comment (alanmoo) [reply donate/thunderbird/templates/base.html]:
```suggestion
{% block title_suffix %} | {% trans 'Give to Thunderbird' context 'Page title' %}{% endblock %}
```

Comment (alanmoo) [reply donate/thunderbird/templates/base.html]: Yeah, that makes sense to me.

File: maintenance/maintenance.html (hunk 1 above)
Comment (alanmoo) [reply maintenance/maintenance.html]:
```suggestion
        <title>Mozilla Giving Platform</title>
```

File: maintenance/maintenance.html [hunk 3]
```diff
@@ -4,17 +4,17 @@
...
         <meta name="description" content="I gave to @mozthunderbird today to #freetheinbox. Join me to support communication privacy.">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
```

File: maintenance/maintenance.html [hunk 4]
```diff
@@ -4,17 +4,17 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="twitter:card" content="summary_large_image">
-        <meta name="twitter:title" content="Donate to Thunderbird | Give to Thunderbird">
+        <meta name="twitter:title" content="Give to Thunderbird">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
        <meta name="twitter:title" content="Mozilla Giving Platform">
```

File: maintenance/maintenance.html [hunk 5]
```diff
@@ -4,17 +4,17 @@
...
         <meta name="twitter:description" content="I gave to @mozthunderbird today to #freetheinbox. Join me to support communication privacy.">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
```

File: maintenance/maintenance.html [hunk 6]
```diff
@@ -4,17 +4,17 @@
...
         <meta name="twitter:image" content="https://assets.mofoprod.net/images/thunderbird_team.jpg">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
```

File: maintenance/maintenance.html [hunk 7]
```diff
@@ -4,17 +4,17 @@
...
         <meta property="og:type" content="website">
         <meta property="og:url" content="https://give.thunderbird.net/en-US/">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
```

File: maintenance/maintenance.html [hunk 8]
```diff
@@ -4,17 +4,17 @@
...
-        <meta property="og:title" content="Donate to Thunderbird | Give to Thunderbird">
-        <meta property="og:site_name" content="Donate to Thunderbird">
+        <meta property="og:title" content="Give to Thunderbird">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
        <meta property="og:title" content="Mozilla Giving Platform">
```

File: maintenance/maintenance.html [hunk 9]
```diff
@@ -4,17 +4,17 @@
...
+        <meta property="og:site_name" content="Give to Thunderbird">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
        <meta property="og:site_name" content="Mozilla Giving Platform">
```

File: maintenance/maintenance.html [hunk 10]
```diff
@@ -4,17 +4,17 @@
...
         <meta property="og:description" content="I gave to @mozthunderbird today to #freetheinbox. Join me to support communication privacy.">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion

```

File: maintenance/maintenance.html [hunk 11]
```diff
@@ -4,17 +4,17 @@
...
         <meta property="og:image" content="https://assets.mofoprod.net/images/thunderbird_team.jpg">
```
Comment (alanmoo) [inline maintenance/maintenance.html]:
```suggestion
```

**Comment by alanmoo** [comment]
@TheoChevalier Can you take a look at this and make sure I didn't miss anything from an l10n standpoint/confirm that these suggestions make sense (since Mavis is out the rest of this week)?

**Pull request merged**